content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -553,3 +553,172 @@ $ curl \
   }
 }
 ```
+
+
+## Associate an SSH key to a workspace
```

Comment:
This wording is odd, though I'm a little hard-pressed to say exactly why. Anyway: in [the related UI docs](https://www.terraform.io/docs/enterprise/workspaces/ssh-keys.html), I used "assign [an SSH key] to," but I think "associate [...] with" would also work; I'm fine with either. 

(And "dissociate [...] from" doesn't have the same problem. ENGLISH.)

Reply:
I'll go with `associate .... with` in this context since `dissociate` is being used. I notice in the UI it talks about `provide` rather than `assign`. I'd be happy to make changes across the board so that the terminology is the same if you'd prefer that.